packages/api-openfin-demo/package.json

```diff
@@ -8,19 +8,26 @@
     "launch": "openfin --launch --config ./src/app.json",
     "start": "npm-run-all --parallel serve launch",
     "postinstall": "rimraf src && copyfiles -f \"node_modules/containerjs-api-specification/src/**\" src && copyfiles -f \"node_modules/containerjs-api-openfin/dist/**\" src",
-    "test": "echo \"No test specified\" && exit 0"
+    "test": "exit 0",
+    "test:ui": "mocha test/main.spec.js"
   },
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
+    "assert": "^1.4.1",
```

Comment:
there are lot of additional dependencies being added to the various projects. Could you just add the test-specific dependencies to `containerjs-api-tests`? these dependencies should propagate through to the projects that depend on `containerjs-api-tests`